robosuite/controllers/composite/composite_controller.py

```diff
@@ -228,6 +228,8 @@ def __init__(self, sim: MjSim, robot_model: RobotModel, grippers: Dict[str, Grip
         # task space actions (such as end effector poses) to joint actions (such as joint angles or joint torques)
 
         self._whole_body_controller_action_split_indexes: OrderedDict = OrderedDict()
+        self._latest_all_joint_angle_action: Optional[np.ndarray] = None
```

Comment:
Can we move the joint angle action related code to some different robosuite branch? E.g. maybe v1.5-joint-actions. For this PR, maybe just have the asset/robot modifications?

Reply:
Removed, will create in another PR

Reply:
perfect